<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>481</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Tilt Network Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_18" rowstretch="0,0,0,0,0,0,0,0" columnstretch="0,0,0,0,0,0,0">
    <item row="7" column="0" colspan="7">
     <layout class="QHBoxLayout" name="stationInfoLayout"/>
    </item>
    <item row="6" column="0" colspan="7">
     <widget class="FrdmViewerWidget" name="oglw" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="connection_but">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QGroupBox" name="LED_gb">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>LED control</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Station</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="stationSelector_cb"/>
         </item>
         <item>
          <widget class="QCheckBox" name="r_checkb">
           <property name="text">
            <string>R</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="g_checkb">
           <property name="text">
            <string>G</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="b_checkb">
           <property name="text">
            <string>B</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="send_pb">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="COM_gb">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>COM config</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="port_cb"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Baud rate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="baudrate_cb">
           <item>
            <property name="text">
             <string>115200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>57600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>38400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2400</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1147</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <widget class="QMenu" name="menuModel">
     <property name="title">
      <string>Model</string>
     </property>
     <addaction name="actionFRDM_K64F"/>
     <addaction name="actionPlane"/>
    </widget>
    <addaction name="actionRefresh_ports"/>
    <addaction name="menuModel"/>
    <addaction name="actionToggle_theme"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuActions"/>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad_2">
   <property name="text">
    <string>Load...</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
  <action name="actionSet_size">
   <property name="text">
    <string>Set size</string>
   </property>
  </action>
  <action name="actionReal_time_Filters">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Real-time Filters</string>
   </property>
  </action>
  <action name="actionAuto_tab_switch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto tab switch</string>
   </property>
  </action>
  <action name="actionRefresh_ports">
   <property name="text">
    <string>Refresh ports</string>
   </property>
  </action>
  <action name="actionFRDM_K64F">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FRDM K64F</string>
   </property>
  </action>
  <action name="actionPlane">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Plane</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionToggle_theme">
   <property name="text">
    <string>Toggle theme</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrdmViewerWidget</class>
   <extends>QWidget</extends>
   <header>src.widgets.frdm_viewer_widget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
